codebundles/k8s-pvc-healthcheck/runbook.robot

```diff
@@ -0,0 +1,137 @@
+*** Settings ***
+Documentation       This taskset collects information about perstistent volumes and persistent volume claims to 
+...    validate health or help troubleshoot potential issues.
+Metadata            Author    Shea Stewart
+Metadata            Display Name    Kubernetes Persistent Volume Healthcheck
+Metadata            Supports    Kubernetes,AKS,EKS,GKE,OpenShift
+Library             BuiltIn
+Library             RW.Core
+Library             RW.CLI
+Library             RW.platform
+Library             OperatingSystem
+Library             DateTime
+Library             Collections
+
+Suite Setup         Suite Initialization
+
+
+*** Keywords ***
+Suite Initialization
+    ${kubeconfig}=    RW.Core.Import Secret
+    ...    kubeconfig
+    ...    type=string
+    ...    description=The kubernetes kubeconfig yaml containing connection configuration used to connect to cluster(s).
+    ...    pattern=\w*
+    ...    example=For examples, start here https://kubernetes.io/docs/concepts/configuration/organize-cluster-access-kubeconfig/
+    ${kubectl}=    RW.Core.Import Service    kubectl
+    ...    description=The location service used to interpret shell commands.
+    ...    default=kubectl-service.shared
+    ...    example=kubectl-service.shared
+    ${KUBERNETES_DISTRIBUTION_BINARY}=    RW.Core.Import User Variable    KUBERNETES_DISTRIBUTION_BINARY
+    ...    type=string
+    ...    description=Which binary to use for Kubernetes CLI commands.
+    ...    enum=[kubectl,oc]
+    ...    example=kubectl
+    ...    default=kubectl
+    ${CONTEXT}=    RW.Core.Import User Variable    CONTEXT
+    ...    type=string
+    ...    description=Which Kubernetes context to operate within.
+    ...    pattern=\w*
+    ...    example=my-main-cluster
+    ${NAMESPACE}=    RW.Core.Import User Variable    NAMESPACE
+    ...    type=string
+    ...    description=The name of the namespace to search.   
+    ...    pattern=\w*
+    ...    example=otel-demo
+    ...    default=''
```

Comment:
`default=` should work, otherwise you'll get `"\'\'"` - users should be specifying a namespace anyways though so not really a problem